build.gradle

```diff
@@ -12,20 +12,51 @@ repositories {
     jcenter()
 }
 
+def embulkVersion = '0.10.31'
+
 group = "com.treasuredata.embulk.plugins"
-version = "0.3.10"
+version = "0.3.11"
 description = "Loads records From Zendesk"
 
 sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
 dependencies {
-    compileOnly  "org.embulk:embulk-core:0.10.18"
+    compileOnly "org.embulk:embulk-api:$embulkVersion"
+    compileOnly "org.embulk:embulk-spi:$embulkVersion"
+
+    compile("org.embulk:embulk-util-config:0.3.1") {
+        // Conflict with Embulk Core
+        exclude group: 'com.fasterxml.jackson.core', module: 'jackson-annotations'
+        exclude group: 'com.fasterxml.jackson.core', module: 'jackson-core'
+        exclude group: 'com.fasterxml.jackson.core', module: 'jackson-databind'
+        exclude group: 'com.fasterxml.jackson.datatype', module: 'jackson-datatype-jdk8'
+        exclude group: 'javax.validation', module: 'validation-api'
+    }
+    compile 'org.embulk:embulk-util-retryhelper:0.8.1'
+    compile "org.embulk:embulk-util-json:0.1.0"
+
+    // embulk 10.31 has problem with null value
+    compile "org.embulk:embulk-guess-csv:0.10.36"
+    compile "org.embulk:embulk-parser-csv:0.10.36"
+    compile "org.embulk:embulk-input-file:$embulkVersion"
```

Comment:
Do we need them in `main`, not `test`?

Reply:
It is redundant let me remove it. Thanks